ccd-definition/FixedLists/EmploymentTypeCheckboxFixedListLRspec.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
```

Comment:
Instead of using checkbox or FixedList in the name I would call it something like EmploymentStatusOptions or similar, to avoid using names of controls/types in the name.

Reply:
this is helping me with readability have a look at this complex type
```
[
  {
    "ID": "EmploymentTypeComplexTypeDQLRspec",
    "ListElementCode": "employmentTypeComplexTypeDQLRspec",
    "FieldType": "MultiSelectList",
    "FieldTypeParameter": "EmploymentTypeCheckboxFixedListLRspec",
    "ElementLabel": "Employed or self-employed?",
    "HintText": "Select both if your client is employed and self-employed.",
    "SecurityClassification": "Public"
  }
]
```

Reply:
As in the previous case, lets discuss naming convention with the whole team, maybe in a retro